app/models/SecurityUser.scala

```diff
@@ -37,7 +37,9 @@ case class CachedUser(
   email: String,
   isActive: Boolean,
   lockStatus: String
-)
+) {
+  def fullName = s"$firstName $lastName"
```

Comment:
Could probably be a val? Not that i think it'll hold this PR up - miniscule efficiency different